README.md

```diff
@@ -587,7 +587,21 @@ class TritonPythonModel:
       # inference_request = pb_utils.InferenceRequest(model_name='model_name',
       #   requested_output_names=['REQUESTED_OUTPUT_1', 'REQUESTED_OUTPUT_2'],
       #   inputs=[<list of pb_utils.Tensor objects>],
-      #   request_id="1", correlation_id=4, model_version=1)
+      #   request_id="1", correlation_id=4, model_version=1, flags=0)
+
+      # The `flags` argument can be used to indicate whether the request is starting
```

Comment:
Wouldn't it be better to document sequence model in BLS outside the python model example? Let't keep this example simple. And describe all this in a separate section below.